Rakefile

```diff
@@ -1,10 +1,3 @@
-require 'airbrake/tasks'
```

Comment:
Probably best to `require 'govuk_app_config` here too, so that when a rake task is added in the future it'll have Sentry enabled automatically.

Reply:
Done